xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -561,7 +569,7 @@ class CPVRRecording final : public CVideoInfoTag
   std::string m_parentalRatingCode; /*!< Parental rating code */
   std::string m_parentalRatingIcon; /*!< parental rating icon path */
   std::string m_parentalRatingSource; /*!< parental rating source */
-
```

Comment:
Please do not delete the line which works as a logical separator between the different member types. Keep an empty line before `mutable CCriticalSection m_critSection;`, please.

Reply:
I didn't know that, done.